src/browser/replay/recorder.js

```diff
@@ -75,6 +75,10 @@ export default class Recorder {
 
     recordingSpan.setAttribute('rollbar.replay.id', replayId);
 
+    if (occurrenceUuid) {
+      recordingSpan.setAttribute('occurrence.uuid', occurrenceUuid);
```

Comment:
Since this is an attribute on the span itself and not an event attribute for our custom events, I think it makes sense to prefix. Sorry for the runaround there, i'll also update mine